src/components/blocks/table/Table.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ComponentProps } from 'react';
+import cn from 'classnames';
+import Html from 'src/components/blocks/Html';
+import { HtmlComponentProps, ValidReactElement } from 'src/components/html-component-props';
+import HtmlDataTypes from '../../../../data/types/html';
+
+import { table, tableContainer } from './Table.module.css';
+
+type Props = {
```

Comment:
I think this can just be replaced with: `HtmlComponentProps<'table'>`

Reply:
I see that this causes a TS error, which we should handle imo; if data is null or `isString` (using lodash or lodash/fp), we should return 'null' as this would be an invalid table.